src/Keystore/StoredKey.cpp

```diff
@@ -0,0 +1,115 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "StoredKey.h"
+
+#include <nlohmann/json.hpp>
+#include <fstream>
+#include <iostream>
+
+using namespace TW;
+using namespace TW::Keystore;
+
+StoredKey StoredKey::load(const std::string& path, const std::string& password) {
+    std::ifstream stream(path);
+    nlohmann::json j;
+    stream >> j;
+
+    StoredKey key(j);
+    return key;
+}
+
+void StoredKey::store(const std::string& path, const std::string& password) {
+    auto stream = std::ofstream(path);
+    stream << json();
+}
+
+// -----------------
+// Encoding/Decoding
+// -----------------
+
+namespace CodingKeys {
+    static const auto address = "address";
+    static const auto type = "type";
+    static const auto id = "id";
+    static const auto crypto = "crypto";
+    static const auto activeAccounts = "activeAccounts";
+    static const auto version = "version";
+    static const auto coin = "coin";
+}
+
+namespace UppercaseCodingKeys {
+    static const auto crypto = "Crypto";
+}
+
+namespace TypeString {
+    static const auto privateKey = "private-key";
+    static const auto mnemonic = "mnemonic";
```

Comment:
should we consider storing `watch` or `address` types too? as just an address

Reply:
ok